paint.js

```diff
@@ -622,11 +624,21 @@ PaintCanvasMorph.prototype.undo = function () {
     }
 };
 
+PaintCanvasMorph.prototype.disableSmoothing = function(ctx) {
```

Comment:
Well, and I don't want any workarounds that build up layers of cruft that we have to keep in the back of our minds. disableSmoothing should be fine